api/locations/index.ts

```diff
@@ -8,9 +8,9 @@ import { EnhancedRequest } from '../lib/models';
 import { LocationTypeEnum } from './data/locationType.enum';
 import { SERVICES_COURT_TYPE_MAPPINGS } from './data/serviceCourtType.mapping';
 import { LocationModel } from './models/location.model';
-import { CourtVenue } from '../workAllocation2/interfaces/location';
-import { handleLocationGet } from '../workAllocation2/locationService';
-import { prepareGetSpecificLocationUrl } from '../workAllocation2/util';
+import { CourtVenue } from '../workAllocation/interfaces/location';
+import { handleLocationGet } from '../workAllocation/locationService';
+import { prepareGetSpecificLocationUrl } from '../workAllocation/util';
```

Comment:
Minor comment - import ordering incorrect here '.' should be below '..'

Reply:
Thanks for comment Connor, purpose of this pr is removing and replacing workallocation2 not finetuning, because if we have  a problem later we can trace it s because of removing MVP or other development. So for tracing purpose it s the best with minimal touch and only focusing scope of story.